command/kv_helpers.go

```diff
@@ -64,6 +67,15 @@ func kvPreflightVersionRequest(client *api.Client, path string) (string, int, er
 			return "", 1, nil
 		}
 
+		// if the original request had the -output-curl-string or -output-policy flag,
+		if (currentOutputCurlString || currentOutputPolicy) && resp.StatusCode == 403 {
+			// we provide a more helpful error for the user,
+			// who may not understand why the flag isn't working.
+			err = fmt.Errorf(
+				`For KV commands, this output flag requires the success of a 
```

Comment:
Do we need to mention "For KV commands" here? I'm assuming this error will only happen for KV commands.

Reply:
Yeah I was on the fence about this one too so I'll remove it.